src/test/tpkg/060-symbols.tpkg/060-symbols.test

```diff
@@ -11,4 +11,6 @@ cp -p ${SRCROOT}/src/libgetdns.symbols libgetdns.symbols.orig
 )
 cp -p ${SRCROOT}/src/libgetdns.symbols libgetdns.symbols.new
 cp -p libgetdns.symbols.orig ${SRCROOT}/src/libgetdns.symbols
-diff libgetdns.symbols.orig libgetdns.symbols.new
+sort libgetdns.symbols.orig > libgetdns.symbols.temp
+diff libgetdns.symbols.temp libgetdns.symbols.new
+
```

Comment:
I have a better fix for this that makes sure the output of mk-symfiles.sh will be the same on all different systems.
The trick is to put LC_ALL=C before the sort command therein.